tests/flow_server_tests.yaml

```diff
@@ -0,0 +1,13 @@
+# This file describes the responses to HTTP requests made
+# to the flow server
+
+# TODO: add more test cases
+
+app: flow_server
```

Comment:
you're not testing the streamed data?

Reply:
Yes. My understanding is that this file should be used for test cases for the flow server, unless this label is for something else that I'm not aware of.

Should I change it to something more specific? e.g. `app: stream_flows`

Reply:
I think Pavel means that you're not consuming the whole stream, just one chunk. Now that I think about it, you do need to consume the stream so you can check the final response.

Reply:
Got it. I'll add the expected stream values and test against them 😄